docs/docs/api/styled.md

```diff
@@ -6,15 +6,15 @@ sidebar_label: styled
 
```

Comment:
This file had a conflict, so when the pre-commit formatting run happened, it was included. No changes to this file that are important.